### ISSUE TITLE
Support class static blocks (Stage-3)

### ISSUE THREAD

**Issue by kitsonk** [denoland/deno#11854]
A Stage-3 proposal to allow static initialisation blocks: https://github.com/tc39/proposal-class-static-block

- [x] v8 tracking bug: https://bugs.chromium.org/p/v8/issues/detail?id=11375
- [x] Shipping in TypeScript 4.4: https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#static-blocks #11678 
- [ ] swc issue: https://github.com/swc-project/swc/issues/2125
  - [x] https://github.com/swc-project/swc/issues/2195
- [x] deno fmt issue: https://github.com/dprint/dprint-plugin-typescript/issues/273

**Comment by F3n67u** [comment]
Seems deno don't support class static blocks now.  wonder when will this feature support by deno?

```
➜  cat mod.ts                                
class C {
  static field1 = console.log('field 1');
  static {
    console.log('static block 1');
  }
  static field2 = console.log('field 2');
  static {
    console.log('static block 2');
  }
}
➜  deno run mod.ts 
error: Unexpected token `{`. Expected identifier, string literal, numeric literal or [ for the computed key at file:///deno_test/mod.ts:3:9
➜  deno --version
deno 1.13.2 (release, x86_64-apple-darwin)
v8 9.3.345.11
typescript 4.3.5
```

**Comment by bartlomieju** [comment]
It will be released in 1.14 on Sept 14th
